File: cpp/problem2.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <cmath>
#include <vector>

using namespace std;

void printbill(vector<string> & items, vector<float> & prices, vector<int> & discounts)
{
    ;
}

int main()
{
    int n;
    cin >> n;

    vector<string> items;
    vector<float> prices;
    vector<int> discounts;

    for (int i = 0; i < n; ++i) {
        cin >> items[i] >> prices[i] >> discounts[i];
    }

    printbill(items, prices, discounts);

    return 0;
}

File: python/problem2.py
def print_bill(items, prices, discounts):
    #print(items, prices, discounts)
    print("Total", 0.00)



n = int(input())
items = []
prices = []
discounts = []
for i in range(n):
    line = input()
    (item, price, discount) = line.split();
    items.append(item)
    prices.append(price)
    discounts.append(discount)

print_bill(items, prices, discounts)
